kbase-extension/static/kbase/js/widgets/function_output/kbaseAttributeMapping.js

```diff
@@ -5,7 +5,8 @@ define([
     'kbwidget',
     'jquery',
     'narrativeConfig',
-    'kbaseAuthenticatedWidget',
+    'kb_service/client/workspace',
```

Comment:
Once other things were fixed, this started failing due to an expectation that there would be a globally available Workspace client. Now it uses the Promisified one from `kb_service`